client/web/src/enterprise/campaigns/list/CampaignsListEmpty.tsx

```diff
@@ -0,0 +1,214 @@
+import React from 'react'
+
+export interface CampaignsListEmptyProps {
+    // Nothing for now.
+}
+
+export const CampaignsListEmpty: React.FunctionComponent<CampaignsListEmptyProps> = () => (
+    <div className="web-content">
+        <h2 className="my-5">Get started with campaigns</h2>
+        <h3 className="mb-3">Tutorials to help with your first campaign</h3>
+        <div className="row">
+            <div className="col-12 col-md-6 mb-2">
+                <div className="card h-100 p-2">
+                    <div className="card-body d-flex">
+                        <FindReplaceIcon className="mr-3" />
+                        <div>
+                            <h4>
+                                <a
+                                    href="https://docs.sourcegraph.com/campaigns/tutorials/search_and_replace_specific_terms"
+                                    rel="noopener"
+                                >
+                                    Finding and replacing exclusionary terms
+                                </a>
+                            </h4>
+                            <p className="text-muted mb-0">
+                                A Sourcegraph query plus a simple sed command creates changesets required to manage a
+                                large scale refactoring.
```

Comment:
Changing text is not a refactoring. Using the wrong word here can get us an immediate 👎 from other developers.

```suggestion
                                large scale change.
```